Solutions/Corvus.Extensions.Newtonsoft.Json.Specs/JsonExtensionsSpecsSteps.cs

```diff
@@ -32,7 +32,7 @@ public JsonExtensionsSpecsSteps(FeatureContext featureContext, ScenarioContext s
         }
 
         [Given(@"I set a property called ""(.*)"" to the value ""(.*)""")]
-        public void GivenISetAPropertyCalledToTheValue(string propertyName, string value)
+        public void GivenISetAPropertyCalledToTheValue(string propertyName, string? value)
```

Comment:
Could this ever be null? My understanding is that if you had a binding `I set a property called "name" to the value ""` then you'd get an empty string passed for the `value` parameter.

Reply:
Yeah - not obvious here but it's called by another method which explicitly passes null as an argument (https://github.com/corvus-dotnet/Corvus.Extensions.Newtonsoft.Json/blob/master/Solutions/Corvus.Extensions.Newtonsoft.Json.Specs/JsonExtensionsSpecsSteps.cs#L41-L45)